utils/cliutils/commandsflags.go

```diff
@@ -1117,6 +1119,14 @@ var flagsMap = map[string]cli.Flag{
 		Name:  excludePatterns,
 		Usage: "[Optional] Filter deployed artifacts by setting a wildcard pattern that specifies which artifacts to exclude. You may provide multiple patterns separated by ', '.` `",
 	},
+	enableSnapshots: cli.BoolFlag{
+		Name:  enableSnapshots,
+		Usage: "[Optional] Set to true to enable snapshot resolution.` `",
```

Comment:
```suggestion
		Usage: "[Default: true] Set to false to disable snapshot resolution.` `",
```
Boolean flags always have a default value, and their usage should describe the opposite functionality.